### ISSUE TITLE
Sugerencia : Ciudades

### ISSUE THREAD

**Issue by loorlab** [nhocki/pico-y-placa-medellin#3]
Muy buena iniciativa, como el dominio es tan generalizado picoyplaca.org , deberían tener para todas las ciudades que tienen pico y  placa en Colombia.

Saludos !

LOOR Lab

**Comment by nhocki** [comment]
Sí. He tenido ganas de ponerle más ciudades (que detecte por IP dónde está uno) pero no he tenido tiempo...

Si está interesado en una ciudad específica, podrías hacer fork del repo y puedo poner un subdominio (ciudad.picoyplaca.org) a apuntar ahí.

**Comment by loorlab** [comment]
@nhocki Listo vamos a hacer un Fork y te enviamos un avance para Bogotá. Gracias por la respuesta.

**Comment by Ondama** [comment]
Acabo de ver que la página del tiempo tiene [API](http://www.eltiempo.com/pico-y-placa/data-by-city/Medellín/2017-10-12)
El link que puse tiene la información de Medellín, pero la API funciona también para Bogotá, Bucaramanga y Cali.
